### ISSUE TITLE
General questions. Version control.

### ISSUE THREAD

**Issue by lazarusA** [CiscoAI/amla#32]
This just works for python3?
For what tensorflow versions does it work?

In Enas the micro search is still in progress?

**Comment by tremblerz** [comment]
Yes, the entire framework is using python3. The tensorflow version on which it was built is 1.8. If I remember correctly it also works well with tf versions ranging from 1.4-1.8.
We can help out if you want python2 or Tensorflow 2.0 based support.

The search part of "micro search" of Enas is not in progress but you can run the final discovered model using configs present in the master repository.

**Comment by lazarusA** [comment]
Thank you. Basically, I'm interested in running DARTS and ENAS for a current project I'm working on. But, the infrastructure in my current organisation just works for `tensorflow 1.12.0` and `python 2.7.12`. If you have code that properly works on these frameworks that will be really helpful and worth a `citation`.  :D

**Comment by tremblerz** [comment]
Sure, would be happy to help out. Just need clarification whether you want to use the `search` algorithm of DARTS and ENAS, or the `final model` discovered and reported in their corresponding paper?

**Comment by lazarusA** [comment]
I want the whole thing. I want to apply the `search` algorithms to find the best architectures. First, to see how the results are for CIFAR or even MNIST, and after that to my particular case(own data). For ENAS, I already tried the macro search (with the original code, from the paper), but sadly at some points breaks (version issues), so I didn't get to the micro search :(.

**Comment by lazarusA** [comment]
And to be honest, if someone else already have the proper code running, I think it will be a waste of time reimplement everything again from scratch.

**Comment by tremblerz** [comment]
At this moment we do not have the whole algorithm of ENAS and DARTS in this repository.
